app/controllers/SignInController.scala

```diff
@@ -32,6 +32,6 @@ class SignInController @Inject()(val appConfig: FrontendAppConfig,
                                 (implicit mcc: MessagesControllerComponents) extends FrontendController(mcc) with AuthRedirects {
 
   val signIn: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(toGGLogin(appConfig.ggSignInContinueUrl))
+    Future.successful(Redirect(appConfig.ggSignInContinueUrl))
```

Comment:
Sorry could we change it to something like this:
Redirect(config.ggLoginUrl, Map("continue_url" -> Seq(config.HomePage), "origin" -> Seq(config.appName)))

Then then change application.conf accordingly